scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/Configuration.java

```diff
@@ -21,6 +21,24 @@
 
     private String version;
 
+    private int managementApiPort;
+
+    public static double getCpus() {
+        return CPUS;
+    }
+
+    public static double getMem() {
```

Comment:
I addressed this bytes issue here: https://github.com/mesos/elasticsearch/issues/128